dracut/modules.d/99kiwi-dump-reboot/kiwi-dump-reboot-system.sh

```diff
@@ -50,7 +54,12 @@ function boot_installed_system {
     if getargbool 0 rd.kiwi.debug; then
         boot_options="${boot_options} rd.kiwi.debug"
     fi
-    kexec -l /run/install/boot/*/loader/linux \
+    switchs=""
+    if [ "$(get_kernel_version_majorminor)" -ge "317" ];then
+	# this variant is secure boot capable, but only available with Kernel 3.17+
+	switchs="-s"
```

Comment:
and be more clear on what -s is with the long form

```
options="--kexec-file-syscall"
```

who know what other call types we will have to handle in the future and I think the code is then a bit better prepared for future changes

Thoughts ?